src/style.css

```
@@ -0,0 +1,44 @@
+
+body {
+  margin: inherit;
+}
+
+.stars {
+  display: flex;
+  flex-direction: column;
+  width: 100px;
+}
+
+.star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars div:nth-child(n) {
+  display: flex;
+  justify-content: space-between;
+}
```

давай будемо використовувати класи як селектори, а не теги
```suggestion
.stars  {
  display: flex;
  justify-content: space-between;
}
```